discovery/kubernetes/client_metrics.go

```diff
@@ -46,7 +46,7 @@ var (
 			Help:       "Summary of latencies for HTTP requests to the Kubernetes API by endpoint.",
 			Objectives: map[float64]float64{},
 		},
-		[]string{"endpoint"},
+		[]string{"verb", "endpoint"},
```

Comment:
Why is this verb here and method above? Will it ever not be GET?

Reply:
The labels are merely following the naming patterns of the arguments in the Client-go interfaces.

You do bring up a good point though; it might be prudent to stick to a standardised name for `verb`/`method`

Reply:
I think this should be fixed upstream.